static/app/views/automations/components/conditionsPanel.tsx

```diff
@@ -0,0 +1,130 @@
+import styled from '@emotion/styled';
+
+import {Flex} from 'sentry/components/container/flex';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Action} from 'sentry/types/workflowEngine/actions';
+import type {
+  DataCondition,
+  DataConditionGroup,
+} from 'sentry/types/workflowEngine/dataConditions';
+import {actionNodesMap} from 'sentry/views/automations/components/actionNodes';
+import {dataConditionNodesMap} from 'sentry/views/automations/components/dataConditionNodes';
+
+type ConditionsPanelProps = {
+  actionFilters: DataConditionGroup[];
+  triggers: DataConditionGroup;
+};
+
+function ConditionsPanel({triggers, actionFilters}: ConditionsPanelProps) {
+  return (
+    <Panel>
+      <Flex column gap={space(1)}>
+        <div>
+          {tct('[when:When] any of the following occur', {
+            when: <Badge />,
+          })}
+        </div>
+        {triggers.conditions.map((trigger, index) => (
+          <div key={index}>
+            <Details item={trigger} nodesMap={dataConditionNodesMap} />
+          </div>
+        ))}
+      </Flex>
+      {actionFilters.map((actionFilter, index) => (
+        <div key={index}>
+          <ActionFilter actionFilter={actionFilter} totalFilters={actionFilters.length} />
+        </div>
+      ))}
+    </Panel>
+  );
+}
+
+interface ActionFilterProps {
+  actionFilter: DataConditionGroup;
+  totalFilters: number;
+}
+
+function ActionFilter({actionFilter, totalFilters}: ActionFilterProps) {
+  return (
+    <ActionFilterWrapper showDivider={totalFilters > 1}>
```

Comment:
we only want to show a divider when the automation has multiple action filter blocks like so:
<img width="442" alt="Screenshot 2025-06-03 at 1 56 17 PM" src="https://github.com/user-attachments/assets/ea35af25-eaff-4e59-bf5c-3cff372576b3" />